provisioning/baremetal_webhook.go

```diff
@@ -0,0 +1,165 @@
+package provisioning
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/pkg/errors"
+
+	osclientset "github.com/openshift/client-go/config/clientset/versioned"
+
+	"github.com/openshift/library-go/pkg/operator/resource/resourcemerge"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	admissionregistration "k8s.io/api/admissionregistration/v1"
+	"k8s.io/utils/pointer"
+
+	"github.com/openshift/library-go/pkg/operator/resource/resourceapply"
+
+	"k8s.io/apimachinery/pkg/util/intstr"
+
```

Comment:
can you cleanup the imports into groups please?

Reply:
I relied on goimports but like you said it definitely needs some manual grouping. I fixed it. Thank you @asalkeld.